docs/Definitions.md

```diff
@@ -26,24 +24,14 @@ Definitions and abbreviations for this draft
 
 **Control protocol** Definitions of message formats, data flows, data serializations, and exchange rules for accessing control APIs.
 
-**Control aggregation** Mechanism by which changing one parameter changes a related set of parameters in a defined way.
-
-**Control group** The set of related parameters that is the subject of a control aggregation instance.
-
-**Framework mechanism** Architectural mechanism essential to NCA operation in any type of device. Conformant Devices must implement all of the Framework Mechanisms applicable.
-
-**Feature-set mechanism** Optional mechanisms that may be configured into blockspecs as needed.
+**Framework mechanism** Architectural mechanism essential to NCA operation in any type of device. Conformant Devices implement all of the Framework Mechanisms applicable.
 
-**Sequence** Written **sequence\<V\>:** an ordered set of values of datatype **V** that is accessed via an index. **V** may be a simple datatype or an object.
+**Feature-set mechanism** opt-in mechanisms which devices implement if they implement the specific feature set.
```

Comment:
Difficult one to word without using normative language...
```suggestion
**Feature-set mechanism** optional mechanism implemented by devices that implement a specific feature set.
```

Reply:
I was trying to avoid the word optional because feature sets should be seen as a requirement if devices implement that particular feature set. For example a device wishing to add receiver monitoring as a capability MUST either use the NcReceiverMonitor control class directly or derive it. They should not create their own completely separate model.

Reply:
Understood. Difficult one as opt-in is synonymous with optional.  Would "additional" work better?

Reply:
Recommendation to change wording to:
**Feature-set mechanism** conditional mechanism required when a device implements a specific feature set.